### ISSUE TITLE
Is it supported in Kubernetes Pod --userns=host？

### ISSUE THREAD

**Issue by zhaocheng173** [kubernetes/kubernetes#92803]
**What would you like to be added**:
`docker run --userns=host ` User isolation can be disabled for containers with a single parameter, and is it possible to support it in K8S POD
**Why is this needed**:
In some cases, we may need to go back to the scenario where user isolation is not enabled and give the container larger permissions to test some docker security issues, such as using Docker-Bench-Security to test, the default use of Docker run can be used, whether such parameters can also be specified in POD YAML

**Comment by neolit123** [comment]
please, try the support channels or #sig-node on k8s slack, if you are not getting a response:
https://github.com/kubernetes/kubernetes/blob/master/SUPPORT.md

/sig node

**Comment by Jongy** [comment]
For future reference, in case anyone finds their way here like I did - it seems that k8s doesn't even make use of user namespaces? My playground cluster runs k8s `v1.19.3` and by running `ls -l /proc/*/ns/user`. Of course this all might be relevant to my local settings...

So by using
```
hostPID: true
hostNetwork: true
securityContext:
  runAsUser: 0
  runAsGroup: 0
```
in the Pod spec, and
```
securityContext:
  privileged: true
```
in the Container spec, I got fully privileged containers without the need for a `--userns` equivalent in k8s.
